.travis.yml

```diff
@@ -1,37 +1,64 @@
-language: python
+os:
+  - linux
+  - osx
 
-python:
-    - "2.7"
-virtualenv:
-    # required because we install the python-qt4 ubuntu package
-    system_site_packages: true
+before_install:
+    - if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+         brew update; brew outdated xctool || brew upgrade xctool;
+         pip install virtualenv;
```

Comment:
Could you move thos commands in the install section, just before the other brew calls ?
I think it will be more readable to have every brew call at the same place, and we could remove the before_install section then.